<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Restriccion</class>
 <widget class="QDialog" name="Restriccion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>776</width>
    <height>571</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);
backgroundColourButton-&gt;setStyleSheet(&quot;background-color: red&quot;);</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>721</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Algerian&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Para iniciar sesion, necesitas responder las siguientes preguntas correctamente:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>531</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;Algerian&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>¿Que equipo ganó las finales de la NBA en el año 2016?</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>501</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;Algerian&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>¿Cual es el mayor goleador en la Copa Mundial?</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>631</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;Algerian&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>¿Quien es el numero 1 en el mundo, en cuanto a tenis se refiere?</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>480</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Algerian&quot;;
color: rgb(255, 255, 255);
background-color: rgb(255, 170, 127);</string>
   </property>
   <property name="text">
    <string>Terminar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>380</y>
     <width>261</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;Algerian&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Resuelva la siguiente integral:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>370</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/integral.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>580</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QRadioButton" name="radioButton">
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Algerian&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Cleveland Cavaliers</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_2">
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Algerian&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Golden State Warrios</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_3">
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Algerian&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Miami Heat</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>350</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QRadioButton" name="radioButton_4">
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Algerian&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Ronaldo</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_5">
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Algerian&quot;;
color: rgb(255, 255, 255);
</string>
      </property>
      <property name="text">
       <string>Pelé</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_6">
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Algerian&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Miroslav Klose</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>480</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QRadioButton" name="radioButton_7">
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Algerian&quot;;
color: rgb(255, 255, 255);
</string>
      </property>
      <property name="text">
       <string>Rafael Nadal</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_8">
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Algerian&quot;;
color: rgb(255, 255, 255);
</string>
      </property>
      <property name="text">
       <string>Andy Murray</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_9">
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Algerian&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Novak Djokovic</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>420</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QRadioButton" name="radioButton_10">
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Algerian&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_11">
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Algerian&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>1/4</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
